diff --git a/homebox.module b/homebox.module
index 5ea4846..46a21e9 100644
--- a/homebox.module
+++ b/homebox.module
@@ -395,7 +395,9 @@ function homebox_build($page) {
   $row = 0;
   for ($i = 1; $i <= count($regions); $i++) {
     ksort($regions[$i]);
-    $sum_width += $page->settings['widths'][$i];
+    if (!empty($page->settings['widths'][$i])) {
+      $sum_width += $page->settings['widths'][$i];
+    }
     if ($sum_width > 100) {
       $row++;
       $sum_width = 0;
