diff --git a/cloze.info b/cloze.info
index 2d4327c..ba41a1e 100644
--- a/cloze.info
+++ b/cloze.info
@@ -6,11 +6,4 @@ php = 5.1
 dependencies[] = quiz
 dependencies[] = quiz_question
 files[] = cloze.classes.inc
-files[] = cloze.admin.inc
-
-; Information added by drupal.org packaging script on 2012-10-19
-version = "6.x-1.0-alpha1"
-core = "7.x"
-project = "cloze"
-datestamp = "1350630368"
-
+files[] = cloze.admin.inc
\ No newline at end of file
diff --git a/cloze.module b/cloze.module
index 55fc4d4..14a695b 100644
--- a/cloze.module
+++ b/cloze.module
@@ -9,7 +9,7 @@
  */
 function cloze_help($path, $args) {
   if ($path == 'admin/help#cloze') {
-    return t('This module provides a cloze question type for Quiz.');
+    return t('This module provides a fill in the blanks question type for Quiz.');
   }
 }
 
@@ -19,7 +19,7 @@ function cloze_help($path, $args) {
 function cloze_quiz_question_info() {
   return array(
     'cloze' => array(
-      'name' => t('Cloze question'),
+      'name' => t('Fill in the blanks question'),
       'description' => t('Quiz questions that allow a user to fill-in the missing words.'),
       'question provider' => 'ClozeQuestion',
       'response provider' => 'ClozeResponse',
