diff --git a/apps.manifest.inc b/apps.manifest.inc
index 975572c..7ed972a 100755
--- a/apps.manifest.inc
+++ b/apps.manifest.inc
@@ -274,9 +274,6 @@ function apps_retrieve_app_image($url, $title = FALSE, $alt = FALSE) {
       drupal_mkdir(file_build_uri('apps'));
     }
     $file = file_save_data($request->data, $uri, FILE_EXISTS_RENAME);
-    if ($current) {
-      $file->fid = $current->fid;
-    }
     $file->title = $title ? $title : '';
     $file->alt = $alt ? $alt : $file->title;
     file_save($file);
